Makefile

```diff
@@ -79,15 +79,13 @@ dist: clean ## builds source and wheel package
 	pipenv run tox -etwine
 	ls -l dist
 
-init: ## install the dev environment
+install: ## install the package to the active Python's site-packages
+	python setup.py install
+
+dev: ## set up a development environment
```

Comment:
To depend on 'clean' based on the same reasoning as https://github.com/release-depot/toolchest/pull/43?

Reply:
fixed